src/auth-service/models/Candidate.js

```diff
@@ -69,30 +70,76 @@ CandidateSchema.statics = {
   },
   async list({ skip = 0, limit = 5, filter = {} } = {}) {
     try {
-      let candidates = await this.find(filter)
+      const project = {
+        _id: 1,
+        firstName: 1,
+        lastName: 1,
+        email: 1,
+        description: 1,
+        category: 1,
+        long_organization: 1,
+        jobTitle: 1,
+        website: 1,
+        status: 1,
+        createdAt: 1,
+        updatedAt: 1,
+        existing_user: { $arrayElemAt: ["$user", 0] },
```

Comment:
in the candidate details, also share if there is any existing user with the same email address.